tcms/testcases/forms.py

```diff
@@ -112,6 +112,56 @@ def clean_case_status(self):
         return self.cleaned_data['case_status']
 
 
+class EditCaseForm(forms.ModelForm):
+
+    class Meta:
+        model = TestCase
+        exclude = ['author', 'reviewer', 'tag', 'component', 'plan']
+
+    default_tester = UserField(required=False)
+    requirement = forms.CharField(required=False)
```

Comment:
Do you really need to declare all of the fields? Unless you are changing the widget or some attributes like required/length, I don't think you need to declare them and they will be inherited from the model, no ?